src/modules/AdvancedPaste/AdvancedPaste/Helpers/JsonHelper.cs

```diff
@@ -147,9 +150,9 @@ internal static string ToJsonFromXmlOrCsv(DataPackageView clipboardData)
                         }
 
                         // A CSV line is valid, if the delimiter occurs more or equal times in every line compared to the first data line. (More because sometimes the delimiter occurs in a data string.)
-                        if (line.Count(x => x == delim) >= delimCount)
+                        if (Regex.Count(line, delim + CsvDelimSepRegexStr) == delimCount && int.IsEvenInteger(Regex.Count(line, delim + CsvDelimSepRegexStr)))
```

Comment:
```suggestion
                        if (Regex.Count(line, delim + CsvDelimSepRegexStr) == delimCount && int.IsEvenInteger(line.Count(x => x == '"')))
```

Here we should validate the nunber of quotation marks.

Reply:
Gotcha! I forgot about getting the total `"` count rather than the required ones.